specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -0,0 +1,1841 @@
+model AADToken
+  is OAuth2Auth<[
+    {
+      type: OAuth2FlowType.implicit;
+      authorizationUrl: "https://login.microsoftonline.com/common/oauth2/authorize";
+      scopes: ["https://cognitiveservices.azure.com/.default"];
+    }
+  ]>;
+
+  @doc("Enumeration of supported Conversation tasks.")
+  union AnalyzeConversationTaskKind {
+    string,
+    @doc("Conversation task kind")
+    Conversation: "Conversation",
+  }
+  
+  @doc("Enumeration of supported conversational task results.")
+  union AnalyzeConversationTaskResultsKind {
+    string,
+    @doc("Conversation result task kind")
+    ConversationResult: "ConversationResult",
+  }
+
+@discriminator("kind")
+@doc("The base class of a conversation input task.")
+@resource("language")
+model AnalyzeConversationTask {
+  @key
+  @visibility("read", "create", "query")
+  @doc("The base class of a conversation input task.")
+  kind: AnalyzeConversationTaskKind;
+}
+
+@doc("The base class of a conversation input task result.")
+@discriminator("kind")
+model AnalyzeConversationTaskResult {
+  @doc("The base class of a conversation input task result.")
+  kind: AnalyzeConversationTaskResultsKind;
+}
+
+@doc("The input for a custom conversation task.")
+model ConversationalTask extends AnalyzeConversationTask {
+  @doc("The input for a custom conversation task.")
+  kind: AnalyzeConversationTaskKind.Conversation;
+
+  @doc("The input ConversationItem and its optional parameters")
+  analysisInput: ConversationAnalysisOptions;
+
+  @doc("Input parameters necessary for a Conversation task.")
+  parameters: ConversationTaskParameters;
+}
+
+@doc("The input ConversationItem and its optional parameters")
+model ConversationAnalysisOptions {
+  #suppress "@azure-tools/typespec-azure-core/bad-record-type"
+  @doc("The abstract base for a user input formatted conversation (e.g., Text, Transcript).")
+  conversationItem: ConversationItemBase;
+}
+
+#suppress "@azure-tools/typespec-azure-core/bad-record-type"
+@doc("The abstract base for a user input formatted conversation (e.g., Text, Transcript).")
+model ConversationItemBase is Record<unknown> {
```

Comment:
Why is this Record<unknown>? I very much suspect that this is incorrect w. regards to the actual service behavior...

Reply:
The ConversationItemBase  does not need additional properties. We already extend this object to add extra properties when needed do I will remove it.

Reply:
But this can trigger the breaking change.